.woodpecker/stg.yml

```diff
@@ -0,0 +1,42 @@
+when:
+  - branch: release/*
+    event: push
+
+skip_clone: true
+
+labels:
+  zone: CLOUD
+
+steps:
+  - name: Run stg site update
+    image: itkdev/ansible-plugin:1
+    when:
+      branch: release/*
+      event: push
```

Comment:
This duplicates lines 1–3. Is `when` needed here?

Reply:
Correct. Removed

Reply:
Global `when` prevents steps from running: https://woodpecker-ci.org/docs/usage/workflow-syntax#when---global-workflow-conditions